google/cloud/apigeeregistry/v1/admin_models.proto

```diff
@@ -25,6 +25,71 @@ option java_multiple_files = true;
 option java_outer_classname = "AdminModelsProto";
 option go_package = "github.com/apigee/registry/rpc;rpc";
 
+// BuildInfo describes a server build.
+message BuildInfo {
+  // Version of Go that produced this binary.
+  string go_version = 1;
+
+  // The main package path.
+  string path = 2;
+
+  // A module used to create the build.
+  message Module {
+    // The module path.
+    string path = 1;
+    // The module version.
+    string version = 2;
+    // The module checksum.
+    string sum = 3;
+    // An optional replacement for the module
+    Module replacement = 4;
```

Comment:
Do we need this? It seems to cover a very specific situation where the module containing the main package is replaced with a different module, but I can't imagine why we would ever do that or want to see it in the build info from a running server.

Reply:
Yes (if I understand you), replacing the main module would be weird.

This message is also used to represent dependencies, and I think that's what led to this [field's inclusion in the runtime/debug package](https://cs.opensource.google/go/go/+/refs/tags/go1.17.6:src/runtime/debug/mod.go;drc=987e4e892325ea91a15a34f7a858ab007fea39c4;l=34) - and I think it's useful here to potentially identify dependencies that are introduced/changed with replace statements.